frontend/src/components/ShootWorkers/ContainerRuntime.vue

```diff
@@ -69,22 +62,34 @@ export default {
     kubernetesVersion: {
       type: String,
       required: true
-    },
-    criNameIsRequired: {
-      type: Boolean,
-      default: false
     }
   },
   data () {
     return {
-      validationErrors,
-      valid: undefined
+      valid: undefined,
+      validationErrors
     }
   },
-  validations,
+  validations () {
+    return this.validators
+  },
   computed: {
-    criNames () {
-      return map(this.machineImageCri, 'name')
+    criItems () {
+      const criItems = map(this.machineImageCri, cri => {
+        return {
+          value: cri.name,
+          text: cri.name,
+          valid: true
+        }
+      })
+      if (!includes(criItems, this.criName)) {
```

Comment:
Shouldn't we disable that invalid value?

Reply:
Yes, you are right of course, I changed it to disabled and removed the valid property. However we still have to do the error validation manually as far as I know